test/runtests.jl

```diff
@@ -40,8 +40,16 @@ end
         datadir = joinpath("test", "data")
         # Process a saved set of coverage data...
         r = process_file(joinpath(datadir, "Coverage.jl"))
+
         # ... and memory data
-        analyze_malloc(datadir)
+        malloc_results = analyze_malloc(datadir)
+        if Sys.iswindows()
+            filename = "test\\data\\testparser.jl.9172.mem"
+        else
+            filename = "test/data/testparser.jl.9172.mem"
+        end
```

Comment:
You can just make this
```julia
filename = joinpath(@__DIR__, "data", "testparser.jl.9172.mem")
```
That takes care of the path separator differences across platforms.

Reply:
Yes, that is much cleaner. Updated and force-pushed.